e2e/rbd.go

```diff
@@ -421,6 +421,73 @@ var _ = Describe("RBD", func() {
 				validateRBDImageCount(f, 0, defaultRBDPool)
 			})
 
+			By("create an erasure coded PVC and bind it to an app", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass: %v", err)
+				}
+				err = createRBDStorageClass(
+					f.ClientSet,
+					f,
+					defaultSCName,
+					nil,
+					map[string]string{
+						"dataPool": "ec-pool",
+						"pool":     "replicapool",
+					},
+					deletePolicy)
+				if err != nil {
+					e2elog.Failf("failed to create storageclass with error %v", err)
+				}
+				pvc, app, err := createPVCAndAppBinding(pvcPath, appPath, f, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create pvc and application binding: %v", err)
+				}
+				err = checkPVCDataPoolForImageInPool(f, pvc, defaultRBDPool, "ec-pool")
+				if err != nil {
+					e2elog.Failf("failed to check data pool for image with error %v", err)
+				}
+				err = deletePVCAndApp("", f, pvc, app)
+				if err != nil {
+					e2elog.Failf("failed to delete pvc and application : %v", err)
+				}
+				// validate created backend rbd images
+				validateRBDImageCount(f, 0, defaultRBDPool)
+			})
+
+			By("create an erasure coded PVC and validate snapshot restore", func() {
+				validatePVCSnapshot(
```

Comment:
need image present in ec pool validation for both snapshot and clone

Reply:
Thanks! Added data pool validation for both snapshot and clone :+1: